content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,224 @@ aliases:
+
+For more real-time tracking of the count of custom metrics for a particular metric name, click into the metric name on the [Metrics Summary page][8]; it’s listed as “Currently reporting # distinct metrics…” as shown below:
+
+{{< img src="account_management/billing/custom_metrics/tracking_metric.mp4" alt="Tracking metric" video="true" responsive="true">}}
+
+## Counting Custom Metrics
+
+**A custom metric is uniquely identified by a combination of a metric name and tag values (including the host tag)**:
+
+* Reporting the same metric name on multiple hosts results in multiple custom metrics.
+* Adding tags on a metric can change the number of custom metrics (number of unique tag value combinations) associated with that particular metric. See the [Effect of Adding Tags](#effect-of-adding-tags) section.
+* Reordering of tag values doesn’t add uniqueness, the following combinations are the same custom metric:
+
+  *  `metric_name{env:dev, service:foo}`
+  *  `metric_name{service:foo, env:dev}`
+
+Below are examples of how to count your custom metrics. The number of custom metrics associated with a particular metric name depends on its metric [submission type][9]:
+
+{{< tabs >}}
+{{% tab "Count, Rate, Gauge" %}}
+
+The number of custom metrics from [COUNT][1], [RATE][2], and [GAUGE][3] metric types is calculated the same way as shown below:
+
+Suppose you’re submitting a metric, `request.count`, from two hosts (`host:A`,`host:B`), which counts the number of endpoint requests. You’re submitting this metric with two tag keys:
+
+* `endpoint`  that can take the value `endpoint:X` or `endpoint:Y`
+* `status` that can take the value `status:200` or `status:400`
+
+Let’s assume that in your data, endpoint `endpoint:X` is supported by both hosts, but fails only on `host:B`, and requests to `endpoint:Y` are always successful and only appears on `host:B` as shown below:
+
+{{< img src="account_management/billing/custom_metrics/request_count.png" alt="Custom metrics host" responsive="true" style="width:80%;">}}
+
+`request.Count` reports then **4 distinct custom metrics**. The custom metrics are listed below:
+
+| Metric Name     | Tag Values                           |
+|-----------------|--------------------------------------|
+| `request.Count` | `host:A`, `endpoint:X`, `status:200` |
+| `request.Count` | `host:B`, `endpoint:X`, `status:200` |
+| `request.Count` | `host:B`, `endpoint:X`, `status:400` |
+| `request.Count` | `host:B`, `endpoint:Y`, `status:200` |
+
+
+### Effect of Adding Tags
+
+Adding tags **may not** result in more custom metrics. Your count of custom metrics usually scales with the most granular/detailed tag.
+
+Let’s suppose you’re measuring temperature in the US and you’ve tagged your temperature metric by country and region. You submit the following data to Datadog:
```

Comment:
Let’s suppose you’re measuring temperature in the US and you’ve tagged your temperature metric by `country` and `region`. You submit the following data to Datadog: